src/config.c

```diff
@@ -237,7 +237,7 @@ void cfg_init(char *name, int allow_missing)
 	cfg_name = str_alloc_copy(path_expand(name));
 	file = fopen(cfg_name, "r");
 	if (!file) {
-		cfg_name = str_alloc_copy(path_expand_ex(name));
+		cfg_name = str_alloc_copy(path_expand_ex(basename(name)));
```

Comment:
`basename()` isn't quite portable so we have `jtr_basename()` in in jumbo.h for use here

Reply:
As in 74a386b3775c27f288abf8bcc3a4eab107df0062 [1]? Or should I use `jtr_basename`?

[1] From jumbo.h
```.h
#undef basename
#define basename(a) jtr_basename(a)
```

Reply:
`jumbo.h` is included from `misc.h`

Reply:
Ah, sorry. That is confusing but anyway you're all set. Perhaps we should stop things like that but that's a whole different issue.